src/server/job_recov.c

```diff
@@ -522,437 +517,3 @@ job_recov_fs(char *filename)
 
 	return (pj);
 }
-
-
-/**
- * @brief
- *		Saves (or updates) a job or resc_resv structure
- *		image on disk
- *
- *		Save does either - a quick update for state changes only,
- *			 - a full update for an existing file, or
- *			 - a full write for a new job or reservation file
- *
- *		For a quick update, the data written is less than a disk block
- *		size and no size change occurs.
- *
- *		No need of O_SYNC flag as this will improve the performance.
- *		This might lead to data loss from file system in case of system
- *		crash. This is not an issue as data is mostly recovered from the
- *		database.
- *
- *		For a new file write, first time, the data is written directly to
- *		the file.
- *
- * @param[in] pobj - Pointer to the resv/job obj to save
- * @param[in] updatetype - Type of update, 0 = quick, 1=full, 2=new
- * @param[in] objtype - Type of object
- *						JOB_OBJECT,
- *						RESC_RESV_OBJECT
- *
- * @return     Error code
- * @retval	0 - Success
- * @retval	-1 - Failure
- *
- */
-
-int
-job_or_resv_save_fs(void *pobj, int updatetype, int objtype)
```

Comment:
I have removed them in my PR - so you don't need to do the same and have reviewers review this twice